### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by luisendelgado**
Fixing breakage in integration tests

### CODE REVIEW

**Review by mdmathias — CHANGES_REQUESTED** [review]

File: Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift [hunk 1]
```diff
@@ -152,6 +159,22 @@ extension DaysUntilBirthdayUITests_iOS {
       .typeText(Credential.password.rawValue)
     sampleApp.keyboards.element.buttons["return"].tap()
 
+    if sampleApp
+      .staticTexts[passwordManagerPrompt]
+      .waitForExistence(timeout: timeout) {
+      guard sampleApp
+        .buttons["Not Now"]
```
[the remark this answers is not in the document]
Comment (luisendelgado) [reply Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift:166]: Done

File: Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift [hunk 2]
```diff
@@ -240,4 +260,44 @@ extension DaysUntilBirthdayUITests_iOS {
...
+    sampleApp.buttons["Continue"]
+      .waitForExistence(timeout: timeout)
+
+    if currentlyShowingSignInDisclaimer {
+      sampleApp.buttons["Continue"].tap()
+    }
+  }
+
+  /// This method looks for an account in the current view that reflects an already-signed-in state, and taps it.
+  /// Returns true if the signed-in account was found and tapped, otherwise returns false.
```
[the remark this answers is not in the document]
Comment (luisendelgado) [reply Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift:290]: Done

File: Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift [hunk 3]
```diff
@@ -240,4 +260,44 @@ extension DaysUntilBirthdayUITests_iOS {
...
+  func findAndTapExistingSignedInEmail() -> Bool {
+    guard sampleApp.staticTexts[Credential.email.rawValue].exists else {
+      XCTFail("Email used for previous sign-in not in list")
```
[the remark this answers is not in the document]
Comment (luisendelgado) [reply Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift:293]: Done

File: Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift [hunk 4]
```diff
@@ -240,4 +260,44 @@ extension DaysUntilBirthdayUITests_iOS {
...
+      return false
+    }
+    guard sampleApp.staticTexts[Credential.email.rawValue].isHittable else {
+      XCTFail("Email used for previous sign-in not tappable")
```
[the remark this answers is not in the document]
Comment (luisendelgado) [reply Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift:297]: Done

File: Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift [hunk 5]
```diff
@@ -26,6 +26,7 @@ class DaysUntilBirthdayUITests_iOS: XCTestCase {
   private let additionalAccessHeaderText = "Days Until Birthday wants additional access to your Google Account"
   private let appTrustWarningText = "Make sure you trust Days Until Birthday"
   private let chooseAnAccountHeaderText = "Choose an account"
+  private let nowNowText = "Not Now"
```
Comment (mdmathias) [inline Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift:29]: `notNowText`?

File: Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift [hunk 6]
```diff
@@ -163,12 +164,12 @@ extension DaysUntilBirthdayUITests_iOS {
       .staticTexts[passwordManagerPrompt]
       .waitForExistence(timeout: timeout) {
       guard sampleApp
-        .buttons["Not Now"]
+        .buttons[nowNowText]
         .waitForExistence(timeout: timeout) else {
-        XCTFail("Failed to find 'Not Now' button")
+        XCTFail(String(format: "Failed to find '%@' button", nowNowText))
```
Comment (mdmathias) [inline Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift:169]: I think string interpolation is more succinct: `XCTFail("Failed to find \(notNowText) button.")`.

File: Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift [hunk 7]
```diff
@@ -163,12 +164,12 @@ extension DaysUntilBirthdayUITests_iOS {
...
         return false
       }
-      sampleApp.buttons["Not Now"].tap()
+      sampleApp.buttons[nowNowText].tap()
```
Comment (mdmathias) [inline Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift:172]: `notNowText` here and elsewhere.

File: Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift [hunk 8]
```diff
@@ -287,14 +288,14 @@ extension DaysUntilBirthdayUITests_iOS {
   }
 
   /// This method looks for an account in the current view that reflects an already-signed-in state, and taps it.
-  /// Returns true if the signed-in account was found and tapped, otherwise returns false.
+  /// - returns: true if the signed-in account was found and tapped, otherwise returns false.
   func findAndTapExistingSignedInEmail() -> Bool {
     guard sampleApp.staticTexts[Credential.email.rawValue].exists else {
-      XCTFail("Email used for previous sign-in not in list")
+      XCTFail(String(format: "Could not find %@ in list of previously-used accounts.", Credential.email.rawValue))
```
Comment (mdmathias) [inline Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift:294]: Apologies -- I think I suggested that we interpolate the email in here. Let's not print that out.

File: Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift [hunk 9]
```diff
@@ -287,14 +288,14 @@ extension DaysUntilBirthdayUITests_iOS {
...
       return false
     }
     guard sampleApp.staticTexts[Credential.email.rawValue].isHittable else {
-      XCTFail("Email used for previous sign-in not tappable")
+      XCTFail(String(format: "%@ used for previous sign-in not tappable", Credential.email.rawValue))
```
Comment (mdmathias) [inline Samples/Swift/DaysUntilBirthday/DaysUntilBirthdayUITests(iOS)/DaysUntilBirthdayUITests_iOS.swift:298]: Same here. Let's not print that email out: `"Email used for previous sign-in not tappable."`

**Review by mdmathias — CHANGES_REQUESTED** [review]

**Review by mdmathias — APPROVED** [review]

**Pull request merged**